### ISSUE TITLE
Possible Ubuntu 18.10 compatibility issue

### ISSUE THREAD

**Issue by Eqlipse** [DIGImend/uclogic-tools#6]
Installed Ubuntu 18.10 yesterday, and while trying to get my Huion Kamvas Pro 13 to work, I tried to install uclogic-tools and ran into this error:

```
./bootstrap && ./configure && make

checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for LIBUSB... no
configure: error: Package requirements (libusb-1.0 >= 1.0.0) were not met:

No package 'libusb-1.0' found
```
However, when checking that I have it installed, I noticed that the naming convention is slightly different:
```libusb-1.0-0/cosmic,now 2:1.0.22-2 amd64 [installed,automatic]```

There is no other option for libusb when I search across the repositories, so I assume that the one I have installed is what's needed, but perhaps the compiler is failing to find it because of a slightly different label?

**Comment by spbnick** [comment]
Thank you for the report, @Eqlipse. I tried to reproduce the issue on a fresh install of Ubuntu 18.10, but couldn't. Make sure you install the `libusb-1.0-0-dev` package mentioned in the README.md, though.

**Comment by spbnick** [comment]
Closing now, please reopen if it still doesn't work for you.

**Comment by Eqlipse** [comment]
Ah. I'd like to clarify something about that readme, then...

> Requirements:
libusb >= 1.0
Libusb development packages are usually named libusb-1.0-0-dev or libusbx-devel. Debian-based systems also need the pkg-config package installed. 

It doesn't actually say "You _need_ the development version." Reading the statement, it starts off with libusb **>=1.0** , which means "greater than **or equal to**" version 1.0... then says where you can get versions greater than 1.0. 

In other words, it's worded incorrectly. If 1.0 is not sufficient, then it should say so, instead of saying that 1.0 is fine, but higher versions will also work (which is what the current wording actually says).

**Comment by spbnick** [comment]
Thank you, @Eqlipse, I'll take a look into this.

**Comment by spbnick** [comment]
@Eqlipse, please take a look at the updated README.md and see if the instructions are less confusing now.

The original "libusb >= 1.0" condition referred to the upstream package and version, not to distribution package and version, since that's more universal. Also, to build anything you normally need a distribution's development package installed for the upstream package. All of this might be familiar to developers, but certainly not to everyone. So I removed this general description, and only left the more specific instructions to avoid confusion. Hope that's better.

**Comment by Eqlipse** [comment]
> @Eqlipse, please take a look at the updated README.md and see if the instructions are less confusing now.
> 
> The original "libusb >= 1.0" condition referred to the upstream package and version, not to distribution package and version, since that's more universal. Also, to build anything you normally need a distribution's development package installed for the upstream package. All of this might be familiar to developers, but certainly not to everyone. So I removed this general description, and only left the more specific instructions to avoid confusion. Hope that's better.

@spbnick It is, thank you for the clarification!

**Comment by spbnick** [comment]
OK, cool :smiley:
